main.js

```diff
@@ -2471,44 +2825,52 @@ function startInstance(id, wakeUp) {
                         if (procs[id].errors > 300) {
                             procs[id].errors.splice(procs[id].errors.length - 300);
                         }
-                        cleanErrors(id, now);
+                        cleanErrors(procs[id], now);
                     });
                 }
 
                 storePids(); // Store all pids to make possible kill them all
 
-                !procs[id].startedInCompactMode && procs[id].process && procs[id].process.on('exit', exitHandler);
+                !procs[id].startedInCompactMode && !procs[id].startedAsCompactGroup && procs[id].process && procs[id].process.on('exit', exitHandler);
 
                 if (!wakeUp && procs[id] && procs[id].process && procs[id].config.common && procs[id].config.common.enabled && (!procs[id].config.common.webExtension || !procs[id].config.native.webInstance) && mode !== 'once') {
                     if (procs[id].startedInCompactMode) {
-                        logger.info(`host.${hostname} instance ${instance._id} started in COMPACT mode`);
-                    } else {
-                        logger.info(`host.${hostname} instance ${instance._id} started with pid ${procs[id].process.pid}`);
+                        logger.info(`${hostLogPrefix} instance ${instance._id} started in COMPACT mode`);
+                    }
+                    else if (procs[id].startedAsCompactGroup) {
+                        logger.info(`${hostLogPrefix} instance ${instance._id} is handled by compact group controller pid ${procs[id].process.pid}`);
+                    }
+                    else {
+                        logger.info(`${hostLogPrefix} instance ${instance._id} started with pid ${procs[id].process.pid}`);
                     }
                 }
             } else {
-                !wakeUp && procs[id] && logger.warn('host.' + hostname + ' instance ' + instance._id + ' already running with pid ' + procs[id].process.pid);
+                !wakeUp && procs[id] && logger.warn(hostLogPrefix + ' instance ' + instance._id + ' already running with pid ' + procs[id].process.pid);
             }
             break;
 
         case 'schedule':
+            if (compactGroupController) {
+                logger.debug(hostLogPrefix + ' ' + instance._id + ' schedule is not started by compact group controller');
```

Comment:
Would it make sense to have a special compact group controller for schedules then?

Reply:
currently schedules are not handled by compat at all ... this would be more a "feature request "for later :-)

Reply:
I do not wanted to make it even bigger. In theory oit should be comparable ... but then memory footprint would be bigger because code is loaded also if only running once a day ...